ui/config_loader.py

```diff
@@ -1,9 +1,10 @@
 import os.path
 from ConfigParser import ConfigParser
 
-def load ():
+config_file = 'config.rc'
+def load (config_file = config_file):
```

Comment:
Why is the ability to load anything other than config.rc being added? Not _really_ against it, just curious.